src/org/mozilla/javascript/regexp/NativeRegExp.java

```diff
@@ -109,6 +109,12 @@
 
     private static final int ANCHOR_BOL = -2;
 
+    private static final SymbolKey GET_FLAGS = new SymbolKey("[Symbol.getFlags]");
```

Comment:
Is this a `Symbol`? `Symbol` is written as `[ @@match ]`.
https://262.ecma-international.org/12.0/#sec-get-regexp.prototype.flags